tests/unit/s2n_mutual_auth_test.c

```diff
@@ -159,7 +167,7 @@ int main(int argc, char **argv)
         /* Craft a cipher preference with a cipher_idx cipher
            NOTE: Its safe to use memcpy as the address of server_cipher_preferences
            will never be NULL */
-        memcpy(&server_cipher_preferences, default_cipher_preferences, sizeof(server_cipher_preferences));
+        memcpy_check(&server_cipher_preferences, default_cipher_preferences, sizeof(server_cipher_preferences));
```

Comment:
Should you be using memcpy_check in a unit test? Wouldn't it be more correct to do EXPECT_BLAH(memcpy())?

Reply:
Good catch, will update to EXPECT_SUCCESS(memcpy_check())

Reply:
We shouldn't merge it in as-is, even with the separate issue to track resolution. As-is, if memcpy fails, the test immediately exits without calling FAIL_MSG or END_TEST.

The quick fix is to continue using normal memcpy. We don't check that the call succeeds, but we also don't break the test if it fails.